### PR TITLE
Creates stream.new

### PR DESCRIPTION
**Pull request by dylanjha**
This is meant to be an application that we can use at https://stream.new

1. This project was started from the with-mux-video NextJS example, and with some notable enhancements: https://github.com/vercel/next.js/tree/canary/examples/with-mux-video

Enhancements:

1. drag & drop video file support in the UI
1. **Slackbot moderator**. This application will receive webhooks from mux, and on an `video.asset.ready` event it will POST to a Slack webhook if the environment variable `SLACK_WEBHOOK_ASSET_READY` is set. This is what the slackbot message looks like - it contains a video thumbnail, asset ID and playback ID. The intention of this is to be a light-weight way to moderate content that is uploaded.

moderator_2020-07-16_18-08-16

For handling webhooks, this application also shows an example of doing signature verification in NextJS - there is a slight "gotcha" there to access the raw request body.

TODO:

- [x] README.md update - document how to set up webhooks
- [x] Design/copy review - I only did a very lightweight pass on this
- [x] Deploy to stream.new and set env vars over there in vercel

### CODE REVIEW

**Comment by dylanjha** [comment]
This is deployed to https://stream-new.vercel.app/

It has the slackbot moderator set up and everything. Plus steven's new design.

I started adding "record from browser", but I would like to follow up in a 2nd PR for that one.

**Comment by sstriegs** [comment]
There's some weird height issues happening in chrome with normal browser window sizes
Screen Shot 2020-08-25 at 6 07 04 PM

Screen Shot 2020-08-25 at 6 08 29 PM

**Comment by sstriegs** [comment]
Clicking/tapping Info was meant to be an overlay so you end up back on the same page, right now if you click info and then close you go back home and lose the video detail page you were on

Screen Recording 2020-08-25 at 6 14 17 PM

**Comment by sstriegs** [comment]
None of the videos work for me on mobile (safari) they all just keep spinning after the preparing state

IMG_1736

File: stream.new/README_INTERNAL.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This is deploiyed to vercel
```
Comment (mmcc) [inline stream.new/README_INTERNAL.md:1]: > deploiyed

File: stream.new/components/upload-progress-fullpage.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      startUpload(file);
+    }
+  }, [file]);
+
+  return (
+    <Layout>
+      {
+        (errorMessage || error)
+          ? <div><h1>{(error && 'Error fetching API') || errorMessage}</h1></div>
+          : <div className="percent"><h1>{progress ? `${progress}%` : 0}</h1></div>
```
Comment (mmcc) [inline stream.new/components/upload-progress-fullpage.js:85]: This line means 0 doesn't show `0%`, which is a little confusing.

Comment (dylanjha) [reply stream.new/components/upload-progress-fullpage.js:85]: good catch!

**Comment by dylanjha** [comment]
moved to https://github.com/muxinc/stream.new

**Pull request closed without merging**